doc/visual_testing/reporter_visual_inspection_suite.py

```diff
@@ -332,29 +322,127 @@ def report_nifti_labels_masker():
     report = masker.generate_report()
     report.save_as_html(REPORTS_DIR / "nifti_labels_masker_atlas.html")
 
-    dataset = datasets.fetch_development_fmri(n_subjects=1)
-    func_filename = dataset.func[0]
-    masker.fit(func_filename)
+    data = datasets.fetch_development_fmri(n_subjects=1)
+    masker.fit(data.func[0])
     report = masker.generate_report()
     report.save_as_html(REPORTS_DIR / "nifti_labels_masker_fitted.html")
 
 
+#  %%
+# Adapted from examples/06_manipulating_images/plot_nifti_simple.py
+def report_nifti_masker():
+
+    masker = NiftiMasker(
+        standardize="zscore_sample",
+        mask_strategy="epi",
+        memory="nilearn_cache",
+        memory_level=2,
+        smoothing_fwhm=2,
+        mask_args=dict(upper_cutoff=0.9, lower_cutoff=0.8, opening=False),
```

Comment:
Again, these parameters looks really ad hoc.

Reply:
yup sorry I was testing something, will revert back to the example code it was taken from